### ISSUE TITLE
Google sign in does not work on iOS

### ISSUE THREAD

**Issue by ramsayamarin** [hippogamesunity/SimpleSignIn#15]
Deep Link does not contain valid required params. URL params: { authuser = 0; code = ...

On Android it works fine.

**Comment by hippogamesunity** [comment]
Hello, it works for all other developers. Please read setup steps carefully. If you have any issues, please provide more details and reproduction steps.

**Comment by ramsayamarin** [comment]
Hi, thanks for the reply. The only difference between Android and iOS is that I have to add the URI scheme in build settings for iOS, which I did, and I also check in xcode the info.plist that it is there. So it should work as it already does in Android, and it does launch the google sign in pages, but upon return to the app nothing happens on iOS and I get this message:
Deep Link does not contain valid required params. URL params: { authuser = 0; code = ...

**Comment by hippogamesunity** [comment]
Can you share logs?
OnDeepLinkActivated (GoogleAuth.cs) should log deep links activated.
Another moment is that my asset doesn't contain this error message. Can you share stacktrace as well?

**Comment by ramsayamarin** [comment]
I played with it more and discovered that it does work if, after the failed attempt when I shut down the app and launched it again, then it does login with Google on iOS. I have no issues with the Google sign-in on Android or the Apple sign-in on iOS, but it seems that with the Google sign-in on iOS, it causes some issues with Firebase Auth where it complains, 'User interaction is still ongoing, another view cannot be presented'. I'm currently unable to spend more time figuring this out, so I simply disabled Google Sign-in on iOS. I hope some of this information is of use to you. Thank you.
